src/ui/templates/selected-post-template.tsx

```diff
@@ -0,0 +1,306 @@
+import React from 'react';
```

Comment:
this file is not a template: so many logic located inside & it not looks as smth reusable